cmake/aws-crt-cpp-config.cmake

```diff
@@ -4,6 +4,7 @@ find_dependency(aws-c-http)
 find_dependency(aws-c-mqtt)
 find_dependency(aws-c-cal)
 find_dependency(aws-c-auth)
+find_dependency(aws-c-event-stream)
```

Comment:
Related to other comment, if not merged into aws-c-common shouldn't aws-c-checksums be mentioned here?

Reply:
It's in aws-c-event-stream-config.cmake
https://github.com/awslabs/aws-c-event-stream/blob/master/cmake/aws-c-event-stream-config.cmake
Similar to that we don't mention aws-c-io here.
And we are not using any functions from aws-checksums directly, so, I think it's fine?